src/SFA.DAS.Roatp.CourseManagement.Web/Infrastructure/RouteNames.cs

```diff
@@ -73,5 +73,6 @@ public static class RouteNames
 
         public const string AddProviderContact = "AddProviderContact";
         public const string ConfirmUpdateStandardsFromProviderContactEmailPhone = "ConfirmUpdateStandardsFromProviderContactEmailPhone";
```

Comment:
The route names should be prefixed by the journey AddProviderContactDetails, AddProviderContactConfirmUpdateStandards AddPRoviderContactSelectStandards and so on.